.github/workflows/update-jira-fix-version.yml

```
@@ -0,0 +1,302 @@
+      github.event.pull_request.base.ref != 'master' &&
+      (contains(toLower(github.event.pull_request.title), 'cherry pick') ||
+       contains(toLower(github.event.pull_request.title), 'cherry-pick'))
+    runs-on: ubuntu-latest
+
+    steps:
+
+      # Extract the original PR number from the cherry-pick title.
+      # "Automated cherry pick of #37124" → 37124
+      - name: Extract original PR number
+        id: extract
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)